vagrant/boxes.d/99-local.yaml.example

```diff
@@ -15,50 +31,50 @@ centos8-katello-devel:
         - "--foreman-proxy-content-enable-ostree=true"
         - "--katello-devel-modulestream-nodejs=12"
 
-centos8-luna-demo:
-  box: centos8-stream
+almalinux8-luna-demo:
+  box: almalinux8
   memory: 10240
   ansible:
     playbook: playbooks/luna_demo_environment.yml
 
-centos8-hammer-devel:
-  box: centos8-stream
+almalinux8-hammer-devel:
+  box: almalinux8
   memory: 512
   ansible:
     playbook: 'playbooks/hammer_devel.yml'
     group: 'hammer-devel'
     variables: # defaults in roles/hammer_devel/defaults/main.yml
-      #hammer_devel_server: centos8-katello-devel.custom-domain.example.com
+      #hammer_devel_server: almalinux8-katello-devel.custom-domain.example.com
 
 katello-remote-execution:
-  box: centos8-katello-nightly
+  box: almalinux8-katello-nightly
   installer: --enable-foreman-remote-execution
 
-centos8-proxy-devel:
-    box: centos8-stream
+almalinux8-proxy-devel:
+    box: almalinux8
     ansible:
       playbook: 'playbooks/foreman_proxy_content_dev.yml'
       group: 'foreman-proxy-content'
-      server: 'centos8-katello-devel'
+      server: 'almalinux8-katello-devel'
 
 centos7-katello-client:
   box: centos7
   ansible:
     playbook: 'playbooks/katello_client.yml'
     group: 'client'
     variables:
-      katello_client_server: 'centos8-katello-devel'
+      katello_client_server: 'almalinux8-katello-devel'
 
-centos8-katello-client:
-  box: centos8-stream
+almalinux8-katello-client:
+  box: almalinux8
   ansible:
     playbook: 'playbooks/katello_client.yml'
     group: 'client'
     variables:
-      katello_client_server: 'centos8-katello-devel'
+      katello_client_server: 'almalinux8-katello-devel'
```

Comment:
We could remove this. As far as I know this playbook installs katello-agent on a host.

Reply:
That playbook includes, among other things, this role: https://github.com/theforeman/forklift/blob/master/roles/katello_client/tasks/main.yml

It does a bit more than just installing katello-agent so probably best to leave it alone

Reply:
I think it's outside the scope here probably, but I think that box and role/playbook should be removed. But we can have that discussion separately..